packages/webview_flutter/webview_flutter_android/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 3.4.4
```

Comment:
This should be a minor version bump (e.g. `3.5.0`) since its adding a new feature.